htdocs/install/mysql/migration/20.0.0-21.0.0.sql

```diff
@@ -47,3 +47,6 @@ ALTER TABLE llx_contrat ADD COLUMN localtax2 double(24,8) DEFAULT 0;
 ALTER TABLE llx_contrat ADD COLUMN revenuestamp double(24,8) DEFAULT 0;
 ALTER TABLE llx_contrat ADD COLUMN total_ht double(24,8) DEFAULT 0;
 ALTER TABLE llx_contrat ADD COLUMN total_ttc double(24,8) DEFAULT 0;
+
+ALTER TABLE llx_expedition_package CHANGE dangerous_goods dangerous_goods VARCHAR(20) NOT NULL DEFAULT '0';
```

Comment:
See in head of file the comment
-- To change type of field: ALTER TABLE llx_table MODIFY COLUMN name varchar(60);